pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/FileUploadService.java

```diff
@@ -332,7 +332,7 @@ private void sendNotification(final Handle handle,
                                 .map(FileUpload::getFileName)
                                 .joining(System.lineSeparator()))));
 
-        if (result.hasError()) {
+        if (result.hasFailure()) {
             log.error("Can't send an e-mail", result.getThrown());
             return;
```

Comment:
`hasFailure()` does not guarantee that `hasThrown()` is non-null. This should check if `getError()` is present as well otherwise we are going to be missing key details.